assets/scripts/build_gallery.py

```diff
@@ -321,9 +329,38 @@ def get_expectation_instances(expectations_info):
             )
             expectation_tracebacks.write(traceback.format_exc())
             continue
+        except pydantic.ValidationError as exc:
+            logger.error(exc)
+            continue
+        except IndexError:
+            logger.error(f"Expectation {expectation_name} has invalid test case.")
+        except Exception as exc:
+            logger.error(f"Unexpected error occurred: {exc}")
     return expectation_instances
 
 
+def create_expectation_instance(expectation_class: type[Expectation]) -> Expectation:
```

Comment:
Wow nice job coming up with a solution here. What percent of expectations are getting handled by this? Are we losing anything we had before?

Also I think we should add some explanatory comments around this bad boy and likely make (and link here) a task around coming up with a more robust solution.

Reply:
i'll update the docstring - i actually think this will be a good long-term solution, but we could establish stronger guarantees around `Expectations` always having these example parameters. And we should only have a single way to write `Expectation` tests, preferably in the `tests` dir and not on the `Expectation` itself.

Reply:
I know we close but not 100% aligned on this part when we last talked, but just to document what I'd like to see in the future:
* Not checking 2 places and potentially raising
* Not reaching into dicts for the data or reading from disk
* Strong guarantees and fast feedback when we fail to implement one of these. If we can do that with static typing, that seems ideal, but I suspect that requires adding the example itself to the expectation.
* I'd also like to see this not get pulled from test data. If anything, I think the example instance should live outside of both tests and business logic, and then get pulled into tests and the gallery build.
But none of that's blocking here of course.